### PR TITLE
Add yolink MultiOutlet

### PR DESCRIPTION
**Pull request by matrixd2**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/yolink.markdown [hunk 1]
```diff
@@ -54,3 +54,4 @@ The integration is tested and verified for the following devices from YoLink:
 * YS4002/3 (Thermostat)
 * YS4906/8 YS7706 (Garage Door Sensor/ Garage Door Controller)
 * YS5707 (Dimmer)
+* YS6801/2 (MultiOutlet)
```
Comment (frenck) [inline source/_integrations/yolink.markdown:57]: This PR is missing a description, so I have to ask: Why is this added?

It helps if you take a second just to motivate your PRs.

Thanks 👍 

../Frenck

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/yolink.markdown (hunk 1 above)
Comment (matrixd2) [reply source/_integrations/yolink.markdown:57]: I'm pushing my code to core, Add support for MultiOutlet

Comment (frenck) [reply source/_integrations/yolink.markdown:57]: Please create a core PR before creating documentation. You now have an orphan documentation PR, triggering the bot and my review...

Comment (matrixd2) [reply source/_integrations/yolink.markdown:57]: Sorry, I shoud do that before create this PR.

**Review by frenck — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.
